bfsscp/bfsscp.go

```diff
@@ -144,7 +144,7 @@ func (b *bucket) Glob(ctx context.Context, pattern string) (bfs.Iterator, error)
 	}
 
 	// Need to walk the tree for global pattern
-	if pattern == "**" {
+	if strings.Contains(pattern, "**") {
```

Comment:
Hm... What happens if we have a pattern like `/foo/*/bar`? Probably, makes sense to do a full-bucket walk on any `*` in pattern?

(btw, I'm afraid full bucket walk for SSH may result in errors - trying to descend into dirs with no perms)

Reply:
The `matchesIterator` will handle any cases like that. It calls `client.Glob` which works correctly in all other scenarios.